apps/site/lib/site_web/controllers/fare_controller.ex

```diff
@@ -11,7 +11,7 @@ defmodule SiteWeb.FareController do
 
   @options %{
     geocode_fn: &LocationService.Geocode.geocode/1,
-    reverse_geocode_fn: &Geocode.reverse_geocode/2,
+    reverse_geocode_fn: &LocationService.Geocode.reverse_geocode/1,
```

Comment:
I ended up correcting these references to point to the module you wrote, `LocationService.ReverseGeocode.reverse_geocode/2`.

Reply:
However, totally not opposed to consolidating that function with the `LocationService.Geocode.geocode/1` function into one module that has both functions...